_posts/en/newsletters/2020-11-18-newsletter.md

```diff
@@ -0,0 +1,108 @@
+  added payment batching and [payjoin][topic payjoin] capabilities respectively.
+
+- **Nunchuk open sources Bitcoin Core backed multisig library:**
+  The team that developed the [Nunchuk desktop application][nunchuk website] has
+  [announced `libnunchuk`][libnunchuk blog], a C++ multisig library that leverages
+  Bitcoin Core's existing codebase.
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [C-Lightning 0.9.2rc1][C-Lightning 0.9.2] is a release candidate for
+  the next maintenance version of C-Lightning.  It contains new
+  features, updated options, and bug fixes.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1 repo],
+[Hardware Wallet Interface (HWI)][hwi repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [C-Lightning #4168][] adds the ability for a plugin to specify that a hook be
+  run before or after that of another plugin. Plugin authors wishing to ensure
+  their plugin's relative load ordering in this way should amend their
+  `getmanifest` method's response as shown [here][C-Lightning getmanifest].
+
+- [C-Lightning #4171][] updates the `hsmtool` command with a new
+  `dumponchaindescriptors` parameter that prints the [output script
+  descriptors][topic descriptors] for the keys and scripts used by
+  C-Lightning's onchain wallet.  These descriptors may then be imported
+  into a watch-only wallet to track any onchain transactions made
+  by the LN node.  This feature was [requested][c-lightning
+  #4110] to help improve integration between [BTCPay
+  Server's][] default hot wallet and the optional LN server.
+
+- [Eclair #1599][] makes spending more intelligent when considering
+  sending a [multipath payment][topic multipath payments] to a channel
+  counterparty.  When the receiver shares a direct channel with the
+  spender, the spender knows exactly how much money is available to be
+  sent in that channel.  With this change, up to that amount can be
+  allocated to the initial part of the payment instead of splitting it
+  across multiple paths.  Any remainder that still needs to be sent can
+  still use other paths.
+
+- [LND #4715][] adds a `--reset-wallet-transactions` configuration
+  parameter that will remove all onchain transactions from LND's wallet
+  and then start a rescan of the block chain to repopulate the wallet.
+
+- [BOLTs #808][] adds a warning that nodes must not release their own
+  HTLC preimages unless they're the final receiver of a payment.  This
+  warning may help new implementations avoid the issue that caused
+  CVE-2020-26896 (see [Newsletter #121][news121 cve-2020-26895]).
```

Comment:
Instead of "the issue", it might be neat to remind the reader what the issue was: "the issue with acceptance of non-standard signatures".

Reply:
Edited, but FYI, the issue was the other one from last week (the premature release of preimages).  I confirmed the link goes to the correct issue.

Reply:
Ah oops, thanks for carefully checking.